### DISCUSSION TITLE
How to disable reinstall for a feature for upgrade(Patch upgrade)

### DISCUSSION THREAD

**Discussion by niha31-github** [wixtoolset/issues#6953] [Questions]
Hi Team,

I have a wix project to generate installer as well as patch. Somehow REINSTALL property for one of the feature is is getting enabled for patch upgrade.
How can I stop the feature to reinstall during upgrade?

**Comment by robmen** [comment]
IIRC, I think this most often happens when you violate the [MSIENFORCEUPGRADECOMPONENTRULES](https://learn.microsoft.com/en-us/windows/win32/msi/msienforceupgradecomponentrules)
